src/UI/Components/DictEditor/DictEditor.tsx

```diff
@@ -128,7 +128,9 @@ const Row: React.FC<RowProps> = ({
 const StyledTextInput = styled(TextInput)`
   &&& {
```

Comment:
what is &&& ? Never seen that before. I know how to use a single ampersand... but three in a row?

Reply:
It was also something new for me, generally its solution from styled-components lib to group all styles that need !important value and give them required priority

Reply:
!important... whoo getting chills from it 😅

Reply:
I know, but patternfly doesn't like to get some of its styles overwritten without it :D